### PR TITLE
implements patch-article

### PR DESCRIPTION
**Pull request by YShamm**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -192,3 +192,53 @@ describe("GET /api/articles, gets all articles", () => {
...
+describe("PATCH /api/articles/:article_id", () => {
+  test("vote up an article", () => {
+    const newVotes = { inc_votes: 10 };
+
+    return request(app)
+      .patch("/api/articles/3")
+      .send(newVotes)
+      .expect(200) //its 200 bc we return
+      .then((response) => {
+        console.log(response.body, "res in TEST");
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:205]: remove console.logs before committing

File: __tests__/app.test.js [hunk 2]
```diff
@@ -192,3 +192,53 @@ describe("GET /api/articles, gets all articles", () => {
...
+          votes: 10,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+        expect(response.body.article.votes).toBe(10);
+      });
+  });
+
+  test("status 400; bad request, invalid data type", () => {
+    const newVotes = { inc_votes: "ten" };
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:222]: A great test to add👏

File: __tests__/app.test.js [hunk 3]
```diff
@@ -192,3 +192,53 @@ describe("GET /api/articles, gets all articles", () => {
...
+    const newVotes = {};
+    return request(app)
+      .patch(`/api/articles/3`)
+      .send(newVotes)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("no change in vote");
+      });
+  });
+});
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:263]: Great tests so far! Some test cases to consider: 

What about the path of /api/articles/not-an-id 
or
/api/articles/99999

**Review by Beth-McCormack — COMMENTED** [review]
I've left a few comments, well done so far!

File: __tests__/app.test.js (hunk 1 above)
Comment (two-triangles) [reply __tests__/app.test.js:205]: You still haven't removed the console.log

**Comment by two-triangles** [comment]
Remove console.log and then I am happy for you to merge!

**Pull request merged**